mls-rs/src/group/mod.rs

```diff
@@ -339,14 +325,35 @@ where
                 .map_err(|e| MlsError::CryptoProviderError(e.into_any_error()))
         })?;
 
-        let context = GroupContext::new_group(
+        let context = GroupContext {
             protocol_version,
             cipher_suite,
             group_id,
+            epoch: 0,
             tree_hash,
-            group_context_extensions,
+            confirmed_transcript_hash: vec![].into(),
+            extensions: group_context_extensions,
+        };
```

Comment:
Feels like this should be some sort of constructor function given that we need epoch and confirmed_transcript_hash to be default values?

Reply:
It only appears here, but added a constructor.